soroban-sdk/Cargo.toml

```diff
@@ -61,8 +62,17 @@ sha2 = "0.10.7"
 [features]
 alloc = []
 testutils = ["soroban-sdk-macros/testutils", "soroban-env-host/testutils", "soroban-ledger-snapshot/testutils", "dep:ed25519-dalek", "dep:arbitrary", "dep:derive_arbitrary", "dep:ctor", "dep:soroban-ledger-snapshot"]
-hazmat = []
 docs = []
 
+# Umbrella feature that enables all hazmat sub-features (backwards compatible)
+hazmat = [
+  "hazmat-crypto-secp256k1_recover",
+  "hazmat-crypto-secp256r1_verify",
```

Comment:
I'm not sure about one sub feature for each function, as this list might get bloated quickly (I'm going to add some soon for poseidon/poseidon2). I feel grouping them under `hazmat-crypto` (and another for "non-crypto") is sufficient now, but willing to go with it if you feel strongly.

Reply:
Or maybe even areas, such as `hazmat-crypto-sec256k1` and then `hazmat-crypto-podeidon`. Wdyt? cc @dmkozh